fix: introduce new built-in rule for missing required schema properties (#1412)
Redocly/redocly-cli: .changeset/lemon-onions-approve.md

---
"@redocly/openapi-core": minor
---

> Added new `missing-required-schema-properties` rule to check if any of required schema properties is missing.

Reviewer: ```suggestion
Added new `no-required-schema-properties-undefined` rule to check if each required schema properties is defined.
```
To be consistent with our naming, it should be named like `no-xxxxxxxx`. I proposed `no-required-schema-properties-undefined`.

Author: Fixed as well!

> Added new `no-required-schema-properties-undefined` rule to check if each required schema properties is defined.

Reviewer 2: ```suggestion
Added new `no-required-schema-properties-undefined` rule to check if each required schema property is defined.
```

Author: Fixed, thanks!
